protocol/directory.go

```diff
@@ -2,67 +2,43 @@ package protocol
 
 import (
 	"log"
-	"sync"
 
 	"github.com/coniks-sys/coniks-go/crypto"
 	"github.com/coniks-sys/coniks-go/merkletree"
 )
 
 type ConiksDirectory struct {
-	sync.RWMutex
 	pad *merkletree.PAD
-	tbs map[string]*merkletree.TemporaryBinding
 }
 
 func InitDirectory(policies merkletree.Policies, secretKey crypto.SigningKey,
-	directorSize, capacity uint64) *ConiksDirectory {
+	directorSize uint64) *ConiksDirectory {
```

Comment:
Seems like you're cutting off just one letter from ``directorSize``. If you want to abbreviate, maybe just call it something like ``dirSize``?